ingest/utils/s2s_token_client.py

```diff
@@ -4,18 +4,32 @@
 from .dcp_auth_client import DCPAuthClient
 
 
-class S2STokenClient:
-    def __init__(self, dcp_auth_client: 'DCPAuthClient' = None):
-        self.dcp_auth_client = dcp_auth_client
+class ServiceCredential:
+    def __init__(self, value: dict):
```

Comment:
nitpicking but it's a bit hard to know what's actually needed in the dict. Can you use the dict type hint? https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#built-in-types